flathub.json

```diff
@@ -1,3 +0,0 @@
-{
-  "only-arches": ["x86_64"]
```

Comment:
following documentation for `flathub.json` file https://github.com/flathub/flathub/wiki/App-Maintenance#limiting-the-set-of-architectures-to-build-on , it seems wise to set `only-arches` for these 2 architectures since there is also `i386` existing andwe don't want to support it.